### PR TITLE
fix: rename build arg 'models' to 'additional_models'

### PR DESCRIPTION
**Pull request by parano**
## Checklist:

- [ ] My code follows the bentoml code style, both `./dev/format.sh` and
  `./dev/lint.sh` script have passed
  ([instructions](https://github.com/bentoml/BentoML/blob/master/DEVELOPMENT.md#style-check-and-auto-formatting-your-code)).
- [ ] My change reduces project test coverage and requires unit tests to be added
- [ ] I have added unit tests covering my code change
- [ ] My change requires a change to the documentation
- [ ] I have updated the documentation accordingly

### CODE REVIEW

File: bentoml/bentos.py [hunk 1]
```diff
@@ -65,18 +65,19 @@ def pull(tag: t.Union[Tag, str]):
...
     exclude: t.Optional[t.List[str]] = None,
     env: t.Optional[t.Dict[str, t.Any]] = None,
     labels: t.Optional[t.Dict[str, str]] = None,
+    additional_models: t.Optional[t.List[str]] = None,
     build_ctx: t.Optional[str] = None,
     _bento_store: "BentoStore" = Provide[BentoMLContainer.bento_store],
     _model_store: "ModelStore" = Provide[BentoMLContainer.model_store],
 ) -> Bento:
-    """
+    """build a bento
```
Comment (sauyon) [inline bentoml/bentos.py:79]: Was this intentional?

**Comment by parano** [comment]
cc @aarnphm looks like there is an issue with installing test dependencies `setuptools` missing?

File: bentoml/bentos.py [hunk 2]
```diff
@@ -192,7 +193,7 @@ def predict(request_data: np.ndarray):
     build_ctx = os.getcwd() if build_ctx is None else os.path.realpath(build_ctx)
     svc = load(svc_import_str, working_dir=build_ctx)
 
-    models = [] if models is None else models
+    additional_models = [] if additional_models is None else additional_models
```
Comment (aarnphm) [inline bentoml/bentos.py:195]:
```suggestion
    additional_models = additional_models or []
```

**Comment by sauyon** [comment]
> cc @aarnphm looks like there is an issue with installing test dependencies `setuptools` missing?

Time to `poetry run` everything? ;)

File: bentoml/bentos.py (hunk 2 above)
Comment (sauyon) [reply bentoml/bentos.py:195]: Not a huge fan of hacking `or` like this, even if it would be safe.

**Comment by aarnphm** [comment]
> > cc @aarnphm looks like there is an issue with installing test dependencies `setuptools` missing?
> 
> Time to `poetry run` everything? ;)

Probably,  i will have to setup virtualenv :(

**Pull request merged**